Tests/ConnectLibraryTests/ConnectConformance/AsyncAwaitConformance.swift

```diff
@@ -390,8 +390,52 @@ final class AsyncAwaitConformance: XCTestCase {
                     expectation.fulfill()
                 }
             }
+            XCTAssertEqual(XCTWaiter().wait(for: [expectation], timeout: kTimeout), .completed)
+        }
+    }
+
+    func testFailServerStreamingAfterResponse() async throws {
+        try await self.executeTestWithClients { client in
+            let expectedErrorDetail = Grpc_Testing_ErrorDetail.with { proto in
+                proto.reason = "soirée 🎉"
+                proto.domain = "connect-crosstest"
```

Comment:
Is this correct?

Reply:
It is yeah. This didn't change until the repo was moved. This updates to the commit right before that (follow-up PR will address this).